sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -214,7 +223,7 @@ export class ManagedIdentityCredential implements TokenCredential {
       // and it means that the endpoint is working, but that no identity is available.
       if (err.statusCode === 400) {
         throw new CredentialUnavailableError(
-          "The managed identity endpoint is indicating there's no available identity"
+          `ManagedIdentityCredential: The managed identity endpoint is indicating there's no available identity. Message: ${err.message}`
```

Comment:
This was the root of the CI issue: The changes in this PR made it so now this credential only failed due to the IMDS MSI on CI. Since CI is in Azure, the IMDS endpoint is reachable, so the execution reaches this statement, and this statement didn’t match the format we had for all of the other errors in this credential.